docs/requiredtools/typescript.md

```diff
@@ -7,12 +7,188 @@ parent: Required tools
 - [NodeJS 14](https://nodejs.org/en/)
 - [TypeScript](https://www.typescriptlang.org/) `npm i -g typescript`
 
-## Initializing target projects
+## Target project requirements
 
 Before you can compile and run the target project, you will need to initialize it. After initializing the test project, you will need to add references to the [abstraction](../../abstractions/typescript) and the [authentication](../../authentication/typescript/azure), [http](../../http/typescript/fetch), [serialization](../../serialization/typescript/json) packages from the GitHub feed.
 
-Clone a NodeJS/front end TypeScript starter like [this one](https://github.com/FreekMencke/node-typescript-starter).
+## Creating target projects
+
+> Note: you can use an existing project if you have one, in that case, you can skip the following section.
+
+Execute the following command in the directory you want to create a new project.
+
+```Shell
+git clone https://github.com/FreekMencke/node-typescript-starter.git
+npm i
+```
+
+> Note: in this example the `node-typescript-starter` is used, but you can use any node + typescript template.
+
+## Adding dependencies
+
+If you have not already, you will need to create a `.npmrc` to enable access to the packages in the GitHub package feed. The article on installing the [Kiota command line](../generator/tool.md) tool shows how to do this.
+
```

Comment:
maybe include some of the content here? or add a link? https://github.com/microsoft/kiota/tree/main/authentication/typescript/azure

Reply:
A link to the azure identity? We are already listing the dependencies just below.

Reply:
Perfect! Adding more details here.